index/golioth.json

```
@@ -8,7 +8,7 @@
             "description": "This starter project is a standalone application that includes databases (both stateful and time-based), Over-the-air update capabilities, settings service, remote procedure calls (RPCs)",
             "kind": "template",
             "tags": ["dfu","lte"],
-            "compatibleNcs": ["v2.3.0", "v2.4.1", "v2.5.0", "v2.5.1"]
+            "compatibleNcs": ["v2.3.0", "v2.4.1", "v2.5.0", "v2.5.1", "2.5.2"]
```

```suggestion
            "compatibleNcs": ["v2.3.0", "v2.4.1", "v2.5.0", "v2.5.1", "v2.5.2"]
```
throughout, I think?